src/phoenix/experiments/functions.py

```diff
@@ -279,6 +273,14 @@ async def async_run_experiment(test_case: TestCase) -> ExperimentRun:
             exp_run = replace(exp_run, id=resp.json()["data"]["id"])
         return exp_run
 
+    _errors: Tuple[Type[BaseException], ...]
+    if not hasattr(rate_limit_errors, "__iter__"):
+        _errors = (rate_limit_errors,) if rate_limit_errors is not None else ()
```

Comment:
```suggestion
    if issubclass(rate_limit_errors, BaseException):
        _errors = (rate_limit_errors,)
    elif rate_limit_errors is None:
        _errors = ()
```
Suggestion: This seems more explicit.

Reply:
Whoops, this doesn't work. See other comment.